jakarta.ls/src/main/resources/jax-rs.json

```diff
@@ -25,17 +25,73 @@
       }
     },
     "JAX-RS - new resource method": {
-      "prefix": "jaxrm",
+      "prefix": "jaxrmget",
       "body": [
         "@GET",
         "@Produces(MediaType.TEXT_PLAIN)",
         "public String ${1:methodname}() {",
         "\treturn \"hello\";",
         "}"
       ],
-      "description": "JAX-RS REST resource method",
+      "description": "JAX-RS REST GET resource method",
```

Comment:
I think we can just change the descriptions to
"JAX-RS GET resource method" same with POST/PUT/etc.